snmp/datadog_checks/snmp/data/default_profiles/_generic-ospf.yaml

```diff
@@ -4,8 +4,6 @@ metrics:
       OID: 1.3.6.1.2.1.14.10
       name: ospfNbrTable
     symbols:
-      - name: ospfNbr
-        constant_value_one: true
       - OID: 1.3.6.1.2.1.14.10.1.6
         name: ospfNbrState
```

Comment:
Look like after the change `snmp.ospfNbrState` metric won't have `neighbor_state ` tag anymore. 

Ideally, we shouldn't remove tags from existing metrics since it can lead to breaking change.

It seems that https://github.com/DataDog/integrations-core/pull/15077 is already part of 7.47 that will be release soon.

See here at branch 7.47.x: https://github.com/DataDog/integrations-core/blob/7.47.x/snmp/datadog_checks/snmp/data/default_profiles/_generic-ospf.yaml

So, maybe it's too late to remove `neighbor_state` from `snmp.ospfNbrState`.

But that said, we can still move ospfNbr constant value one to a separate table, that will make it easier to add new states/status in the future if needed.